src/pages/Accounts.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div class="content">
+    <card>
+      <template slot="header">
+        <h5 class="title">Accounts</h5>
+        <p class="category">A component to interact with your accounts</p>
+      </template>
+      <div class="row">
+        <div class="col-lg-12 col-md-3 col-sm-4 col-xs-6 col-xs-6 text-left">
+          <card>
+            <p v-for="(account, address) in accounts">{{ account }}. Account: {{ address }}</p>
+            <hr />
+            <p>
+              Choose an address to convert to:
+              <select
+                v-model="selectedChain"
+                @change="onSelect($event)"
+              >
+                <option v-for="chainname in chains" v-bind:value="chainname">{{ chainname }}</option>
+              </select>
+            </p>
+            <hr />
+            <p
+              v-if="selectDropdown()"
+              v-for="(account, address) in accounts"
+            >{{ account }}. Address on {{ onlyChain }}: {{ address | prefix(onlyChain) }}</p>
+          </card>
+        </div>
+      </div>
+    </card>
+  </div>
+</template>
+<script>
+import { Card } from "@/components/index";
+import { web3Accounts } from "@polkadot/extension-dapp";
+import { MyFunctions } from "../components/Filters/Filter.js";
+import api from "../connection.js";
+const util = require("@polkadot/util-crypto");
+let dappex = require("@polkadot/extension-dapp");
+
+export default {
+  el: "#selector",
+  components: {
+    Card,
+    web3Accounts
+  },
+  data() {
+    return {
+      selected: false,
+      accounts: {},
+      address: {},
+      onlyAddress: "",
+      selectedAddress: "",
+      selectedChain: "",
+      onlyChain: "",
+      chainName: "",
+      chains: [
```

Comment:
Keep the DRY principle in mind. Consider grabbing this from the filter instead. To get it from the filter, we would need to:

- add a function to the filter that exports the list
- this function would be consumed by the filter itself, and by any code that imports the filter